### PR TITLE
add data set icon

### LINKED ISSUE #11: Instead of color coding an asset by the marketplace it's hosted on; come up with a more user-friendly alternative
## Scope

- See that e.g. OCEAN data sets, we color pink
- Big data protocol data sets, we color green.

Screen Shot 2021-11-08 at 15 34 19

- From our UX designer, we got the following feedback:

Screen Shot 2021-11-08 at 15 35 39

## Deliverable

- Make information on where the data set was published available in a different way but color-coding the data asset in the table
- Options can be:
  - show the marketplace's icon next to the link
  - show text "ocean protocol" somewhere
  - Add a tag label

## Time estimate

- Discussion here on GitHub is included in the total working time
- A simple proposal can take up to 4 hours of time.

### PR DESCRIPTION
**Pull request by il3ven**
Fixes #11 

Added a icon next to the data sets to convey the protocol. 

Imo the green and red color should not be used. My suggestion is to use black. The other text in the table should have normal font weight and the content should have bold. I couldn't change color from CSS.

image

### CODE REVIEW

File: design.css [hunk 1]
```diff
@@ -835,3 +836,24 @@ thead {
...
+  min-width: 20px;
+  max-width: 20px;
+  height: 20px;
+  background: url("/bdp-logo.png");
+  background-size: contain;
+  background-repeat: no-repeat;
+  margin-left: 0.5rem;
+}
+
+.content {
```
Comment (TimDaub) [inline design.css:855]: Can we please rename this to something descriptive as e.g. "protocol-logo"?

Comment (il3ven) [reply design.css:855]: The content inside `<td>` is wrapped inside a div of this class. This `<td>` is the second column of the table and its heading is Content. That's why I have used it.

```html
<div class="content">
    <div>
      <a
        style="color: #ff4092"
        target="_blank"
        href="https://market.oceanprotocol.com/asset/did:op:7Bce67697eD2858d0683c631DdE7Af823b7eea38"
      >
        Annotated Images
      </a>
    </div>
    <div class="ocean-icon" title="Ocean Protocol"></div>
</div>
```

If the naming still doesn't feel right I'd be happy to change it.

Comment (TimDaub) [reply design.css:855]: ah ok sorry I my bad. Then it's a separate issue.

**Review by TimDaub — APPROVED** [review]
looks great, thanks @il3ven

**Pull request merged**